crates/primitives/state_update/src/lib.rs

```diff
@@ -100,6 +100,7 @@ impl StateDiff {
 
         let storage_diffs_sorted = {
             let mut storage_diffs = self.storage_diffs.clone();
+            storage_diffs.iter_mut().for_each(|storage_diff| storage_diff.sort());
```

Comment:
Consider renaming this function to something more obvious, like `sort_storage`

Reply:
fixed!